yarn.lock

```diff
@@ -9954,6 +9959,15 @@ postcss-selector-parser@^3.0.0, postcss-selector-parser@^3.1.0:
     indexes-of "^1.0.1"
     uniq "^1.0.1"
 
+postcss-selector-parser@^5.0.0-rc.3:
```

Comment:
@btraut these look like the yarn lock file changes I was talking about when I was trying to update gestalt. It seems others are also experiencing them

Reply:
Gotchya. I have no idea why I don't see the issue, but if this fixes it, I'm happy you updated it.